sdk/aqueduct/decorator.py

```diff
@@ -142,7 +143,17 @@ def op(
             A list of relative paths to files that the function needs to access.
             Python classes/methods already imported within the function's file
             need not be included.
-
+    
+    Returns:
+        A decorated function that once called with artifact input(s) will be
+        executed in the cloud and returns a TableArtifact.
+
+            Attributes:
+                local:
+                    A function that once called will be executed locally that conduct
+                    the same operations as the decorated function. The function takes in
+                    either TableArtifact or Dataframe, and return a Dataframe.
```

Comment:
I feel like this description might be a bit hard for users to understand (and the cloud part is not necessarily correct, the user could be running it all locally). Also, we seem to already describe the "return" part in the first few lines of the docstring.

What about just something like (get rid of the headers and just stick this above "Args" on line 137:

```
To run the wrapped code locally, without Aqueduct, use the `local` attribute. Eg:
>>> compute_recommendations.local(customer_profiles, recent_clicks)
```